draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2682,18 +2682,19 @@ https://video.example.com/k/movie1/4/013.ts
 
         </list></t>
 
-        <t>Object specification<list style="empty">
-
-              <t>Property: locations<list style="empty">
-
-                 <t>Description: An Access List that allows or denies (blocks) the trigger execution
-                    per cache location.</t>
+	<t>Object specification:</t>
+        <t><list style="empty" hangIndent="3">
+        <t><list style="hanging" hangIndent="3"><t hangText="Name: locations">
```

Comment:
Property vs. Name: So this was "Property" before, and you converted it to "Name".  It appears that "Property" and "Name" are used fairly interchangeably throughout the document.  Should we look at making this more consistent throughout the document?

Reply:
Sure, we can do this in the final PR
I have been aligning the field encoding to this standard where I saw it, don't care either way about Name or Property

Reply:
Marking this as resolved in this PR, made a note to myself to do format alignment in the last PR